paper/paper.md

```diff
@@ -0,0 +1,449 @@
+asked for review from a different developer.
+The reviewer then provided feedback and approved the pull request to be merged
+into the main branch after the writer made any requested changes.
+This way, more than one person viewed each notebook before it could be
+incorporated into the public curriculum, which reduced mistakes and ensured
+higher quality content. <!-- TODO: cite U-M Carpentries paper for the development model once it's submitted to bioRxiv?-->
+While teaching from the curriculum at the first Data Science Summer Experience,
+instructors took notes on their experience and made revisions afterward.
+Maintainers continue to monitor the repository and resolve issues as they arise.
+
+Following the onset of the COVID-19 pandemic, we quickly pivoted our club to a
+virtual format.
+In preparation for the 2020 Summer Experience, we switched to a flipped
+classroom style following feedback from our club participants that it was too
+difficult to follow along live coding via Zoom.
+We recorded facilitators teaching the lesson notebooks as if they were live
+coding, then shared them along with a link to the lesson notebooks for students
+to code along with while watching the videos.
+Each video shows the Jupyter notebook alongside the facilitator themselves
+teaching.
+This format allowed students to learn at their own pace, then ask questions and
+practice when we met together virtually. <!-- TODO: should this actually go later in the curriculum section?-->
+
+Our Jupyter notebooks and links to the lesson videos can be accessed on GitHub:
+https://github.com/GWC-DCMB/curriculum-notebooks
+
+# Curriculum
+
+Our curriculum was designed for high school students with no prior coding
+experience who are interested in learning Python programming for data science.
+However, this course material would be useful for anyone interested in learning
+basic programming for data analysis, regardless of age or experience level.
+
+Our curriculum features short lessons to deliver course material in “bite sized”
+chunks, followed by practices to solidify the learners' understanding.
+
+## Learning Objectives
+
+Our learning objectives are:
+
+1. Understand fundamental concepts and best practices in coding.
+2. Apply data analysis to real world data to answer scientific questions.
+3. Create informative summary statistics and data visualizations in Python.
+
+These skills provide a solid foundation for basic data analysis in Python and
+participation in our programming exposes students to the many ways coding and
+data science can be used to make large impacts across many disciplines.
+
+## Course Content
+
+Our curriculum design consists of 27 modules that cover Jupyter notebook setup,
+Python coding fundamentals, use of essential data science packages including
+Pandas and numpy, basic statistical analyses, and plotting using seaborn and
+matplotlib (Figure 1) [@harris_array_2020; @waskom_seaborn_2021;
```

Comment:
Are we going to include a figure? @marlenad did you ever get around to drafting one? I can also draft one and/or we can just not have a figure.

Reply:
How is this?
![curriculum_fig](https://user-images.githubusercontent.com/22454290/119025986-aaecfb80-b972-11eb-8fd6-1e956e7a0bc2.png)

Reply:
Looks great!

Reply:
super super minor critique: could we pick different colors that aren't the Microsoft default? Or maybe make all the boxes the same color? Could pick a hex value from one of our logos.

Reply:
lol ya sure, let me make up a prototype

Reply:
![curriculum_fig2](https://user-images.githubusercontent.com/22454290/119037113-8f3c2200-b97f-11eb-9fbe-4acb665d054c.png)
this uses the teal colors from the GWC design guide https://girlswhocode.com/assets/downloads/GWC_DesignGuide_Short_z3-2.pdf

Reply:
looks great, thanks @marlenad!